plugin/pkg/scheduler/algorithm/BUILD

```diff
@@ -29,10 +29,18 @@ go_library(
 
 go_test(
     name = "go_default_test",
-    srcs = ["scheduler_interface_test.go"],
+    srcs = [
+        "scheduler_interface_test.go",
+        "types_test.go"
+    ],
     library = ":go_default_library",
     tags = ["automanaged"],
-    deps = ["//pkg/api/v1:go_default_library"],
+    deps = [
+        "//pkg/api/v1:go_default_library",
+        "//vendor:k8s.io/apimachinery/pkg/apis/meta/v1"
+	    "//vendor:k8s.io/kubernetes/pkg/api/v1"
```

Comment:
Here I'm a little confused, it this same as "//pkg/api/v1:go_default_library" ? If so then I won't need this.